pkg/shp/cmd/build/create.go

```diff
@@ -63,6 +63,11 @@ func (c *CreateCommand) Run(params *params.Params, io *genericclioptions.IOStrea
 
 	flags.SanitizeBuildSpec(&b.Spec)
 
+	// print warning with regards to source bundle image being used
+	if b.Spec.Source.BundleContainer != nil && b.Spec.Source.BundleContainer.Image != "" {
+		fmt.Fprintf(io.Out, "Build %q uses a source bundle image, which means source code will be transferred to a container registry. It is advised to use private images to ensure the security of the source code being uploaded.", c.name)
```

Comment:
Looks like this atm:

```bash
$ shp build create test-bundle --source-bundle-image ... --source-credentials-secret cr-sascha --output-image ... --output-credentials-secret
cr-sascha --strategy-name buildkit -n build-test
Build "test-bundle" uses a source bundle image, which means source code will be transferred to a container registry. It is advised to use private images to ensure the security of the source code being uploaded.Created build "test-bundle"
```

Therefore:

```suggestion
		fmt.Fprintf(io.Out, "Build %q uses a source bundle image, which means source code will be transferred to a container registry. It is advised to use private images to ensure the security of the source code being uploaded.\n", c.name)
```